.github/workflows/build-image.yml

```diff
@@ -18,6 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: hadolint/hadolint-action@v3.0.0
```

Comment:
Other lintting is running with [.github/workflows/linter.yml](https://github.com/mastodon/mastodon/blob/48e136605a30fa7ee71a656b599d91adf47b17fc/.github/workflows/linter.yml).

see https://github.com/github/super-linter#readme

Reply:
I know, but running the linter there doesn’t prevent the image from being built, which is what I’d like to avoid when the linter fails because it takes such a long time to build the image)

I could integrate the image build process with the linter, however, that would require some major rearchitecting of the Actions setup and I rather chose the lesser evil (an Action that adds roughly 5 seconds to a process that takes 40-50 minutes on average) than to throw around the entire Actions setup